### PR TITLE
Feature add minor changes

### PR DESCRIPTION
**Pull request by NikolaM-Dev**
## :nerd_face:  Description

1. Change `Gophers-Latam` to `Gophers LATAM`
2. Remove white spaces
3. Add missing space after a period
4. Remove unnecessary word `en`
5. "Uncomment" Go Playground section and change required paragraphs
6. Confirm that 4 is right, _Actually, I don't know where did you take that screenshot_

## :thinking: Questions

1. Why don't we have all the sessions active in the code, but we do have them on the deployed page?
2. Not only do we translate, but we also add extras?

## 🤩 Solves

#23

### CODE REVIEW

File: tour/content/basics.article [hunk 1]
```diff
@@ -14,13 +14,6 @@ El programa que se muestra a la derecha está usando los paquetes con rutas de i
 
 Por convención, el nombre del paquete es el mismo que el último elemento de la ruta de importación. Por ejemplo, el paquete `"math/rand"` comprende archivos que comienzan con la declaración `package`rand`. En cambio `"math"` viene siendo parte de la ruta donde se encuentra el paquete y a su vez tambien puede ser un paquete que comprende archivos que comienzan con la declaración `package`math`.
 
-#appengine: *Nota:* El entorno en el que se ejecutan estos programas es
-#appengine: determinista, por lo que cada vez que ejecute el programa de ejemplo
-#appengine: `rand.Intn` devolverá el mismo número.
-#appengine:
-#appengine: (Para ver un número diferente, inicie el generador de números; ver [[https://golang.org/pkg/math/rand/#Seed][`rand.Seed`]].
-#appengine: El tiempo es constante en el playground, por lo que deberá usar otra cosa como semilla.)
-
```
Comment (NikolaM-Dev) [inline tour/content/basics.article:23]: Deleting deprecated information

File: tour/content/welcome.article [hunk 2]
```diff
@@ -50,15 +50,15 @@ El tour está disponible en otros idiomas:
 - [[https://go.dev/tour/][English - English]] (versión original)
 - [[https://go-tour-br.appspot.com/][Brazilian Portuguese — Português do Brasil]]
 - [[https://go-tour-ca.appspot.com/][Catalan — Català]]
-- [[https://tour.go-zh.org/][Simplified Chinese — 中文（简体）]]
 - [[https://go-tour-cz.appspot.com/][Czech — Česky]]
 - [[https://go-tour-id2.appspot.com/][Indonesian — Bahasa Indonesia]]
 - [[https://go-tour-jp.appspot.com/][Japanese — 日本語]]
 - [[https://go-tour-ko.appspot.com/][Korean — 한국어]]
+- [[https://go-tour-lat.appspot.com/][Spanish — Español]]
```
Comment (NikolaM-Dev) [inline tour/content/welcome.article]: Change to use alphabetic order

File: tour/content/welcome.article [hunk 3]
```diff
@@ -50,15 +50,15 @@ El tour está disponible en otros idiomas:
...
 - [[https://go-tour-pl1.appspot.com/][Polish — Polski]]
 - [[https://go-tour-th.appspot.com/][Thai — ภาษาไทย]]
 - [[https://go-tour-ua-translation.lm.r.appspot.com/][Ukrainian — Українською]]
-- [[https://go-tour-lat.appspot.com/][Spanish — Español]]
+- [[https://tour.go-zh.org/][Simplified Chinese — 中文（简体）]]
```
Comment (ThiagoMowszet) [inline tour/content/welcome.article]: `Simplified Chinese — 中文（简体)` no deberia ir debajo de `Spanish — Español`  si es en orden alfabetico?

**Review by ThiagoMowszet — CHANGES_REQUESTED** [review]
Thanks for the changes @NikolaM-Dev.
Check please if the chinese order is ok!
after that, we can merge.
🖖💯

File: tour/content/welcome.article [hunk 4]
```diff
@@ -54,11 +54,11 @@ El tour está disponible en otros idiomas:
 - [[https://go-tour-id2.appspot.com/][Indonesian — Bahasa Indonesia]]
 - [[https://go-tour-jp.appspot.com/][Japanese — 日本語]]
 - [[https://go-tour-ko.appspot.com/][Korean — 한국어]]
-- [[https://go-tour-lat.appspot.com/][Spanish — Español]]
 - [[https://go-tour-pl1.appspot.com/][Polish — Polski]]
+- [[https://tour.go-zh.org/][Simplified Chinese — 中文（简体）]]
+- [[https://go-tour-lat.appspot.com/][Spanish — Español]]
```
[the remark this answers is not in the document]
Comment (NikolaM-Dev) [reply tour/content/welcome.article:59]: @ThiagoMowszet

Comment (NikolaM-Dev) [inline tour/content/welcome.article:59]: ## Description

1. It was organized based on the hyperlink text (i.e. the alphabetical order of the languages).
2. And it is put in English first, which is the original version, but if you want I can also put English in alphabetical order.

## Preview

image

Comment (NikolaM-Dev) [reply tour/content/welcome.article:59]: @ThiagoMowszet

**Pull request merged**